cmd/eksctl/main.go

```diff
@@ -75,7 +74,6 @@ func main() {
 	rootCmd.SetUsageFunc(flagGrouping.Usage)
 
 	if err := rootCmd.Execute(); err != nil {
-		fmt.Println(err)
```

Comment:
Don't we need to print it anymore?

Reply:
I'm no `cobra` expert and still discovering the various ways to configure it, but my understanding is that, **before** this PR, because we were not propagating errors up to parent commands, we had to:
- `os.Exit(1)` in child commands,
- `fmt.Println(err)` or `logger.Critical("%s\n", err.Error())` ourselves in various places.

In case of an error, for example in during input validation, it would output:
```console
$ eksctl create fargateprofile
Create a Fargate profile

Usage: eksctl create fargateprofile [flags]
[...]
Use 'eksctl create fargateprofile [command] --help' for more information about a command.

[✖]  --cluster must be set
```
----

However, on this branch, **with** `fmt.Println(err)`, because we also propagate the error up, and `cobra` somehow prints it before printing the usage, we end up reporting the error twice:

```console
$ ./eksctl create fargateprofile
Error: --cluster must be set
Usage: eksctl create fargateprofile [flags]
[...]
Use 'eksctl create fargateprofile [command] --help' for more information about a command.


--cluster must be set
```

**Without** `fmt.Println(err)`, we report it just once, as we should:

```console
$ ./eksctl create fargateprofile
Error: --cluster must be set
Usage: eksctl create fargateprofile [flags]
[...]
Use 'eksctl create fargateprofile [command] --help' for more information about a command.
```

N.B.: the description (e.g. "`Create a Fargate profile`") seems also gone though, so I'll have to understand why, and fix it. (If we care, that is, but I think we do.)

Reply:
Apparently not printing the description by default is expected: https://github.com/spf13/cobra#example-1
FWIW, I don't mind if we don't print it. I think it is explicit enough with `Usage: [...]`.
We could fiddle with it and provide custom help/usage message for each command, but I'd rather keep it simple and avoid this for now.
Besides, the description still appears when printing the help/usage, so I think this is more than enough:
```console
$ ./eksctl create fargateprofile --help
Create a Fargate profile

Usage: eksctl create fargateprofile [flags]
[...]
```

Any thought/strong opinion? 🙂